pkg/controllers/managementuser/snapshotbackpopulate/snapshotbackpopulate.go

```diff
@@ -71,13 +86,21 @@ func (h *handler) OnChange(key string, configMap *corev1.ConfigMap) (runtime.Obj
 		return configMap, nil
 	}
 
+	if h.activeConfigMap == "" {
+		h.activeConfigMap = configMap.Name
+	}
```

Comment:
Similarly, can this be put in the `Register` function by getting the cluster object with the index and checking which distribution is being used?

Reply:
We *can*, but I don't know how useful that is. It's an extra hit against the cache, whereas we know we'll only ever process 1 configmap per instance of a cluster with this controller.